src/main/java/org/sonatype/gradle/plugins/scan/ossindex/VulnerabilityUtils.java

```diff
@@ -17,17 +17,19 @@
 
 import java.util.Objects;
 
-public class VulnerabilityUtils
+class VulnerabilityUtils
```

Comment:
Why did the access modifier change?

Reply:
> Why did the access modifier change?

1. The class isn't accessed outside of the `ossindex` package.
2. Make consistent with the other utils (`BannerUtils`) class.